### PR TITLE
remove old HZDR systems

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- hypnos: Is not longer available, the compute resource are now in the
hemera cluster
  - remove also old paraview file for hypnos (you can find it still in old releases)
- hydra: Is still available but not used by PIConGPU users. Due to mixed compute nodes /Main memory settings) and a not maintained software stack it is not useful to keep the system template.

### CODE REVIEW

**Comment by psychocoderHPC** [comment]
> Perhaps we should also remove the references to these systems elsewhere: [1](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/USAGE.rst#tbg), [2](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/share/paraview/hypnos.pvsc), [3](https://github.com/ComputationalRadiationPhysics/picongpu/blob/d64ccf75b5cc8bc08d503d521ecb9fe56a3bf05f/share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg#L49), [4](https://github.com/ComputationalRadiationPhysics/picongpu/blob/d64ccf75b5cc8bc08d503d521ecb9fe56a3bf05f/share/picongpu/examples/FoilLCT/etc/picongpu/8_isaac.cfg#L49)

thx, I will remove it

**Comment by psychocoderHPC** [comment]
@sbastrakov I update the docs and point now to hemera.
The hypnos paraview file is now removed fully, we still have it in older releases but it was outdated anyway and not used.

File: USAGE.rst [hunk 1]
```diff
@@ -101,16 +101,16 @@ tbg
...
 
-For example, if you want to run on the HPC System `"Hypnos" at HZDR <https://www.hzdr.de/db/Cms?pOid=12231>`_, your ``tbg`` submit command would just change to:
+For example, if you want to run on the HPC System `"Hemera" at HZDR <https://www.hzdr.de/db/Cms?pOid=12231>`_, your ``tbg`` submit command would just change to:
 
 .. code-block:: bash
    :emphasize-lines: 2
 
    # request 1 GPU from the PBS batch system and run on the queue "k20"
-   tbg -s qsub -c etc/picongpu/1.cfg -t etc/picongpu/hypnos-hzdr/k20.tpl $SCRATCH/runs/lwfa_002
+   tbg -s qsub -c etc/picongpu/1.cfg -t etc/picongpu/hemera-hzdr/k20.tpl $SCRATCH/runs/lwfa_002
```
Comment (sbastrakov) [inline USAGE.rst:110]: Does `qsub` work on Hemera?

Comment (psychocoderHPC) [reply USAGE.rst:110]: I think so, someone said there is an alias but I will change it to sbatch

File: share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg [hunk 2]
```diff
@@ -46,7 +46,7 @@ TBG_periodic="--periodic 0 0 0"
 ## Section: Optional Variables ##
 #################################
 
-TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hypnos5 --isaac.quality 99"
+TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hemera4 --isaac.quality 99"
```
Comment (PrometheusPi) [inline share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg:49]: This requires that the isaac server will work on hemera - so far, we could only build it for hypnos. @psychocoderHPC Have you compiled the isaac server on hemera?

File: share/picongpu/examples/FoilLCT/etc/picongpu/8_isaac.cfg [hunk 3]
```diff
@@ -46,7 +46,7 @@ TBG_periodic="--periodic 0 0 0"
 ## Section: Optional Variables ##
 #################################
 
-TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hypnos5 --isaac.quality 99"
+TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hemera4 --isaac.quality 99"
```
Comment (PrometheusPi) [inline share/picongpu/examples/FoilLCT/etc/picongpu/8_isaac.cfg:49]: same here

File: share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg (hunk 2 above)
Comment (psychocoderHPC) [reply share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg:49]: This is only an example and hypnos is on the way to get disabled.

Comment (psychocoderHPC) [reply share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg:49]: The bug that it is not building on hemera must be fixed in ISACC.

Comment (psychocoderHPC) [reply share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg:49]: @PrometheusPi We have few open issue in ISAAC if it is a compile issue than please open a new issue there if it is the issue with [dumping png](https://github.com/ComputationalRadiationPhysics/isaac/issues/95) there is no new issue needed.

Comment (PrometheusPi) [reply share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg:49]: I am not sure whether it does not build on hemera. On taurus it build fine but did not work - thus we will have to check this. But this is only a minor adjustment, thus I am fine with changing this to hemera.

**Pull request merged**